tox.ini

```diff
@@ -7,10 +6,11 @@ envlist=
   py35
 
 [testenv]
-deps=
-  -rrequirements.dev.txt
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps= -rtests_require
 commands=
   py.test tests \
     --cov {env:PACKAGE} \
```

Comment:
package name is already repeated so many times in this config... and make does pass it along via the environment.
we could make sure that `make test` also passes `tox` all the command line parameters it got so that we could enjoy both worlds.